src/components/ui/AppChart.vue

```diff
@@ -10,7 +10,7 @@
         ref="chart"
         :option="opts"
         :setOptionOps="{ notMerge: true }"
-        :initOpts="{ renderer: 'svg' }"
+        :init-options="{ renderer: 'svg' }"
```

Comment:
so they remove camel case, but they leave `setOptionsOps`?  lol

Reply:
Odd, I know... but without this change, the build was breaking! After that it worked fine!